jina/executors/__init__.py

```diff
@@ -52,6 +52,8 @@ def __call__(cls, *args, **kwargs):
 
         obj = type.__call__(cls, *args, **kwargs)
 
+        obj.logger = JinaLogger(obj.__class__.__name__, id=m['pod_id'] if 'pod_id' in m else None)
```

Comment:
```suggestion
        obj.logger = JinaLogger(obj.__class__.__name__, id=m.get('pod_id', None))
```

Reply:
where is this `id` used in `JinaLogger`? I don't see any reference

Reply:
Wrong argument name will fix